collect_app/src/main/java/org/odk/collect/android/preferences/AdminPreferencesFragment.java

```diff
@@ -75,7 +83,7 @@ public void onClick(DialogInterface dialog, int which) {
                         dialog.dismiss();
                         Collect.getInstance().getActivityLogger()
                                 .logAction(this, "AdminPasswordDialog", "CHANGED");
-                    } else if (pw.equalsIgnoreCase("") && ver.equalsIgnoreCase("")) {
+                    } else if (pw.equalsIgnoreCase("")) {
```

Comment:
I think there is no need to have a condition to check if `pw` is null, it's already implied as the above condition is false. `else if()` can be replaced with `else`